ports/libsimpleservo/api/src/lib.rs

```diff
@@ -819,12 +819,27 @@ impl EmbedderMethods for ServoEmbedderCallbacks {
             }
         }
 
-        let discovery = webxr::openxr::OpenXrDiscovery::new(
-            Box::new(GlThread(executor)),
-            Box::new(ProviderRegistration(surface_providers)),
-            Box::new(ContextMenuCallback(embedder_proxy)),
-        );
-        registry.register(discovery);
+        if openxr::create_instance().is_ok() {
```

Comment:
Hmm, this would be nicer if we had the prompt when you attempt to enter immersive mode, IMO. Might need another error variant

Reply:
I'll file a followup for that, since it requires a bunch more plumbing.